Diagnostics/DustCollapse/main.cpp

```diff
@@ -105,15 +105,15 @@ int main(int argc, char* argv[])
         auto x_maxdist = fmax(fabs(problo[0]), fabs(probhi[0]));
         auto y_maxdist = fmax(fabs(problo[1] - yctr), fabs(probhi[1] - yctr));
```

Comment:
Should these be updated too?

Reply:
yeah, probably.  I'll do a pass on those and add it to this PR

Reply:
I also see a bunch of places we can use `powi`, but I figure we should do that in another PR

Reply:
yes, that should be separate